app/forms/candidate_interface/section_complete_form.rb

```diff
@@ -0,0 +1,14 @@
+module CandidateInterface
+  class SectionCompleteForm
+    include ActiveModel::Model
+
+    attr_accessor :completed
+    validates :completed, presence: true
```

Comment:
As this is essentially a boolean, should we use the `inclusion` validator rather than presence (as false is a valid option)?

```suggestion
    validates :completed, inclusion: [true, false]
```

Reply:
👍